samples/log_callback/enclave/enc.c

```diff
@@ -1,13 +1,54 @@
 // Copyright (c) Open Enclave SDK contributors.
 // Licensed under the MIT License.
 
+#include <openenclave/tracee.h>
 #include <stdio.h>
+#include <string.h>
 
 // Include the trusted log_callback header that is generated
 // during the build. This file is generated by calling the
 // sdk tool oeedger8r against the log_callback.edl file.
 #include "log_callback_t.h"
 
+void enclave_customized_log(
+    void* context,
+    oe_log_level_t level,
+    uint64_t thread_id,
+    const char* message)
+{
+    char modified_log[200];
+
+    sprintf(
+        modified_log,
+        "E, %s, %llx, %s",
+        oe_log_level_strings[level],
+        thread_id,
+        message);
+
+    /*
+     * Add logic here to modify the log message, to obscure enclave logs from
+     * the host. The context might be used, for example, to transfer a
+     * shared/public/private secret, that may be used to encrypt the log
+     * message.
+     *
+     * NOTE: fprintf does not work consistently in this function. If used, the
```

Comment:
Please open an issue for this bug. I think this can be rephrased as all operations based on OE's host filesystem support would fail.